cylc/flow/cfgspec/suite.py

```diff
@@ -463,7 +463,7 @@
 
             Example:
 
-               ``foo(PT1H30M), bar(PT1.5H), baz``
```

Comment:
Rogue blockquote.

Reply:
I think the whole indentation is off for this config node - should be indented by 4 more spaces. This is what it looks like with this PR branch checked out (ignore the orange highlight, that's just due to ctrl+F)
![Screenshot 2020-12-23 140521](https://user-images.githubusercontent.com/61982285/103003232-51e67c00-4528-11eb-86e0-0309d550c929.png)

Reply:
Ah, looks like I took away one space more than I meant to.

Reply:
It renders correctly now, but the indentation of the `desc` content in the code is 4 spaces less than it should be for consistency.